src/main/java/decodes/cwms/resevapcalc/EvapUtilities.java

```diff
@@ -0,0 +1,716 @@
+/* 
+ * Copyright (c) 2018
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+package decodes.cwms.resevapcalc;
+
+import org.opendcs.units.Constants;
+/**
+ *
+ * @author RESEVAP program by Steven F. Daly (ERDC/CRREL)
+ * conversion to Java by Richard Rachiele (RMA)
+ */
+public class EvapUtilities
+{
+    
```

Comment:
private EvapUtilities() {
        // Prevent instantiation
        throw new IllegalStateException("EvapUtilities class cannot be instantiated");
    }